build.sbt

```diff
@@ -70,7 +70,7 @@ val commonSettings = Seq(
   scalacOptions ++= Seq(
     "-feature",
     "-deprecation",
-    "-unchecked",
-    "-Xlint"
+//    "-Xlint",
```

Comment:
I think you can disable specific warnings although it might still be of a broader scope than you need.